projects/gnu.org/gcc/package.yml

```diff
@@ -69,9 +68,9 @@ build:
     # means we're building with the exact same version FIXME
     - run: |
         ln -sf gcc cc
-        ln -sf gcc-ar ar
-        ln -sf gcc-nm nm
-        ln -sf gcc-ranlib ranlib
+        ln -sf ../../../binutils/v\*/bin/ar ar
```

Comment:
Probs should be a shim to ensure the correct versions are used. But let’s merge for now to see if this works for you or not.

The question becomes: who do gnu provide these if we shouldn't use them?

Reply:
It's a really great question I have no good answer for.